mon-pix/app/services/oidc-identity-providers.js

```diff
@@ -14,6 +15,11 @@ export default class OidcIdentityProviders extends Service {
       .map((provider) => provider.organizationName);
   }
 
+  isIdentityProviderActivated(identityProviderSlug) {
+    const oidcIdentityProviders = this.list.filter((provider) => provider.id === identityProviderSlug);
```

Comment:
Remarque : Un `this[identityProviderSlug]` à la place de `this.list.filter((provider) => provider.id === identityProviderSlug);` fonctionne aussi mais est beaucoup moins explicite